src/app/Http/Controllers/AdminChoiceAddController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\BigQuestion;
+use App\Question;
+use App\Choice;
+
+class AdminChoiceAddController extends Controller
+{
+    public function index(Request $request, $big_question_id, $question_id)
+    {
+        $questions = Question::where('big_question_id',$big_question_id)->get();
+        $param = ['big_question_id' => $big_question_id,'question_id' => $question_id, 'questions' => $questions];
```

Comment:
Question->big_question_idで取得できるのであればbig_question_idは不要ではないかと思いました
```suggestion
        $param = ['question_id' => $question_id, 'questions' => $questions];
```